tooling/noir_js/package.json

```diff
@@ -29,8 +29,8 @@
     "dev": "tsc-multi --watch",
     "build": "tsc-multi",
     "test": "yarn test:node:esm && yarn test:node:cjs",
-    "test:node:esm": "mocha --timeout 25000 --config ./.mocharc.json",
-    "test:node:cjs": "mocha --timeout 25000 --config ./.mocharc.cjs.json",
+    "test:node:esm": "mocha --timeout 25000 --exit --config ./.mocharc.json",
+    "test:node:cjs": "mocha --timeout 25000 --exit --config ./.mocharc.cjs.json",
```

Comment:
Tests currently hang no matter the outcome so I've told mocha to force an exit. See https://mochajs.org/#-exit

Seems like there's an underlying issue which we need to debug in more detail but this at least gets us up and running.

Reply:
Yeah I think it has to do with one of the async operations